.github/workflows/publish-on-release.yml

```diff
@@ -0,0 +1,56 @@
+name: Publish Vanilla release
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build:
+    name: Build Vanilla
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - name: Read Vanilla version from package.json
+        run: |
+          node -p "require('./package.json').version" > VANILLA_VERSION
+          echo "Building Vanilla v$(cat VANILLA_VERSION)"
+      - run: yarn install
+      - run: yarn build
+      - run: yarn test
+      - run: cp VANILLA_VERSION build/css
+      - uses: actions/upload-artifact@v1
+        with:
+          name: css
+          path: build/css
+
+  publish-npm:
+    name: Publish to NPM
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+          registry-url: https://registry.npmjs.org/
+      - run: npm publish
+        env:
+          NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
+
+  publish-assets:
+    name: Publish to assets server
+    needs: build
```

Comment:
Do you think we should wait for `publish-npm` before uploading it to our asset server?

Reply:
Good point. Updated, so we don't upload to assets if npm publish fails.